canonical-websites/www.ubuntu.com
# Update deleted pages and redirects

Fixes #2105

- Add the killed pages from [the IA document](https://docs.google.com/spreadsheets/d/1VhcGdJ4gUoi3U9hd-E8ydgMTxI9iioxamMroUY0woRI/edit#gid=491196604)
- Move some redirects to deleted pages
- Remove other erroneous URLs from redirects (we don't need to be scared of
  404ing stupid URLs. And the Homepage isn't your dumping ground!)

QA
--

`./run`, then check that pages listed in the `deleted.yaml` show up as deleted, that the other redirects still work, and that you agree with my statements here about which pages should be marked as deleted, which should be redirected, and which should be neither.
